src/ngAria/aria.js

```diff
@@ -303,11 +304,20 @@ ngAriaModule.directive('ngShow', ['$aria', function($aria) {
     compile: function(elem, attr) {
       var fn = $parse(attr.ngClick, /* interceptorFn */ null, /* expensiveChecks */ true);
       return function(scope, elem, attr) {
+
+        function isNodeOneOf(elem, nodeTypeArray) {
+          for (var i=0; i < nodeTypeArray.length; i++) {
```

Comment:
Wouldn't the plain old `indexOf` work as well ?

<sub>Also, having spaces around `=` would make jscs happier I believe :)</sub>